widgets/mpd_all.lua

```diff
@@ -43,6 +43,14 @@ function format_progress(elapsed, duration)
 end
 -- }}}
 
+-- {{{ Format playing progress (percentage)
+function format_progress_percentage(elapsed, duration)
+    local percentage = math.floor((elapsed / duration) * 100)
```

Comment:
Do you think it would be more precise if `math.floor((elapsed / duration) * 100 + 0.5)` was used?

Reply:
Fair point, not sure about what approach is more precise.

Reply:
I prefer your version, for example in case of a percentage of 25.9% I like to have 26% instead of 25%.

```
> 200/770
0.2597
> math.floor((200/770)*100)
25
> math.floor((200/770)*100+0.5)
26
```